BaseTools/Source/Python/AutoGen/GenMake.py

```diff
@@ -1196,6 +1198,18 @@ def GetFileDependency(self, FileList, ForceInculeList, SearchPathList):
             Dependency[F] = GetDependencyList(self._AutoGenObject, self.FileCache, F, ForceInculeList, SearchPathList)
         return Dependency
 
+    # MU_CHANGE [BEGIN] - Add Rust build support
+    ## Returns a comma delimited string of enabled cargo features for this module
+    #
+    #   @retval     str            comma delimited string of cargo feature flags
+    #
+    def GetModuleEnabledRustFeatures(self):        
+        features = ""
+        for pcd in self._AutoGenObject.ModulePcdList:
+            if pcd.Type == "FeatureFlag" and bool(pcd.DefaultValue):

```

Comment:
This is just my lack of familiarity with this code. Does `pcd.DefaultValue` include the final PCD not just the default from DEC?

Reply:
I had the same thought when working on this, but I did confirm that overrides outside of the DEC changed the `pcd.DefaultValue`

Reply:
Sounds good, thanks